impressum.md

```diff
@@ -0,0 +1,41 @@
+# Impressum
+
+*Angaben gemäß § 5 TMG*
+
+Brockmeyer Haardiek Kerssen Kerssen Kiesow Rosemann GbR  
+Birkhahnweg 27  
+49124 Georgsmarienhütte
+
+## Vertreten durch:  
+Timo Nogueira Brockmeyer
+
+## Kontakt:
+Telefon:  
```

Comment:
Please just remove the empty field for now. We can add them again once we have the necessary information.

Reply:
will do that.